.coverage.sh

```diff
@@ -22,7 +22,7 @@ jbuilder runtest
 # move all bisect files here
 find . -name 'bisect*.out' -exec mv {} . \;
 
-DIRECTORIES=`find . -type d`
+DIRECTORIES=$(find _build/default -type d -not -path '*/\.*')
```

Comment:
In `'*/\.*'` the `\` is not required but does no harm.

Reply:
I wasn’t sure. Do you want me to remove it?

Reply:
Unless you update this PR for something else, this is fine.